device-to-group-command.py

```diff
@@ -0,0 +1,390 @@
+'''
+esper app usage
+
+usage: esper_group_actions [-h]
+                            -c  {uninstall, whitelist, brightness, alarm_volume, ring_volume, music_volume, notification_volume, bluetooth, wifi, gps, ping, reboot}
+                            -g  GROUP_ID
+                            -v  VALUE
+
+example:
+
+./esper_group_actions -g 52ecfb3c-d1ad-4e66-8cf9-85daff8d7f3c -c ring_volume -v 50
+./esper_group_actions -g 52ecfb3c-d1ad-4e66-8cf9-85daff8d7f3c -c ping
+
+Runs on all active devices in a group. API key is required.
+'''
+
+import sys
+import requests
+import esperclient
+import time
+from esperclient.rest import ApiException
+
+import argparse
+active_device_list = []
+inactive_device_list = []
+
+#### Enterprise Configuration #######
+
+# Please fill the varibales under <>
+configuration = esperclient.Configuration()
+configuration.host = 'https://<endpoint-name>-api.shoonyacloud.com/api'
```

Comment:
Pass `Endpoint-name`, `API-Key` and `Enterprise-ID` as environment variables, while calling the script.
eg
```
$ ENDPOINT=demo API_KEY=blahblah ENTERPRISE=uuid-string esper_group_actions -g 52ecfb3c-d1ad-4e66-8cf9-85daff8d7f3c -c ring_volume -v 50
```

Reply:
This will be okay for developers point of view but we are also using this script with customers as well where we want to ask them to pass minimal information. So that's why keeping inside it only and asked to fill that information in readme